extensions/ql-vscode/package.json

```diff
@@ -182,6 +182,12 @@
           "default": "%q on %d - %s, %r result count [%t]",
           "description": "Default string for how to label query history items. %t is the time of the query, %q is the query name, %d is the database name, %r is the number of results, and %s is a status string."
         },
+        "codeQL.runningTests.additionalTestArguments": {
+          "scope": "window",
```

Comment:
I suggest making this more restrictive: use `machine` just like we do for `codeql.cli.executablePath`. This ensure the setting can only be configured at the user level and prevents malicious workspaces from executing arbitrary code.

Reply:
Is this really a danger in this case? Can the `run test` command invoke arbitrary commands? I guess it would if you can specify a build command for a test database, but not sure if that's possible.

Reply:
Mainly being cautious. Saves us revisiting this in future if we introduce such an option. Right now, I think it's only potentially vulnerable through some indirection, e.g. if the controlled workspace points the test/extractor to a different folder it controls.

Reply:
That's fine by me, probably best to avoid. I just wanted to make sure there wasn't a specific flaw we need to prevent.